app/js/CompileManager.js

```diff
@@ -49,6 +51,79 @@ const getCompileDir = (project_id, user_id) =>
 const getOutputDir = (project_id, user_id) =>
   Path.join(Settings.path.outputDir, getCompileName(project_id, user_id))
 
+function emitPdfStats(stats, timings) {
```

Comment:
Maybe move this function into a module, e.g ContentCacheMetrics.js, to keep the main file smaller?

Reply:
Done in https://github.com/overleaf/clsi/pull/225/commits/22cba9d06f16313307e66fe6175fb2c643288453